hive.tf

```diff
@@ -0,0 +1,23 @@
+/* Hetzner
+ * Dedicated AX41-NVMe Server
+ * Location: Finland, HEL1-DC5
+ * 1 x Primary IPv4
+ * 2 x 512 GB NVMe SSD
+ * 6 Core CPU
+ * 64 GB DDR4 RAM
+*/
+
+module "ethereun_hive_testing" {
```

Comment:
ah fuck

Reply:
should have been `ethereum_hive_testing`

Reply:
You can just re-create it(or use `terraform state mv`) since it's just a dummy resource.

Reply:
yeah just now i re-created it!
missed opportunity for `terraform state mv`